src/commands/stac-setup/__test__/stac.setup.test.ts

```diff
@@ -197,3 +197,15 @@ describe('formatDate', () => {
     assert.equal(formatDate(startYear, endYear), '2023-2024');
   });
 });
+
+describe('formatGsd', () => {
+  it('Should return GSD unaltered', async () => {
+    const gsd = '0.3';
+    assert.equal(formatGsd(gsd), '0.3');
+  });
+
+  it('Should return GSD with trailing m removed', async () => {
+    const gsd = '0.3m';
+    assert.equal(formatGsd(gsd), '0.3');
```

Comment:
NIT: not much point for the variable?

```suggestion
    assert.equal(formatGsd('0.3m'), '0.3');
```